include/mbgl/util/geo.hpp

```diff
@@ -23,17 +23,19 @@ class LatLng {
 public:
     struct null {};
 
-    double latitude;
-    double longitude;
+    double latitude = 0;
+    double longitude = 0;
 
     enum WrapMode : bool { Unwrapped, Wrapped };
 
-    LatLng(null) : latitude(std::numeric_limits<double>::quiet_NaN()), longitude(latitude) {}
+    constexpr LatLng() = default;
 
-    LatLng(double lat = 0, double lon = 0, WrapMode mode = Unwrapped)
+    constexpr LatLng(null) : latitude(std::numeric_limits<double>::quiet_NaN()), longitude(latitude) {}
+
+    constexpr LatLng(double lat, double lon, WrapMode mode = Unwrapped)
```

Comment:
By making `LatLngBounds::world()` constexpr [here](https://github.com/mapbox/mapbox-gl-native/pull/8583/commits/6fce4a65b242fc2d185c787104493b3c0770b511#diff-cf54d0ce1dec44ab05dcd5daf776c7f8R92), `LatLngBounds` needed a constexpr ctor, which required its members (`LatLng` for southwest and northeast bounds) to also have a constexpr ctor. We use`LatLngBounds::world()` to initialize the default `TransformState` bounds [here](https://github.com/mapbox/mapbox-gl-native/pull/8583/commits/4b8849b596fedaaeca4c7437eba9fbb45aa6f776#diff-14cb746fd86755f6100a64ac3c8ee6a9R85).